README.md

```diff
@@ -61,10 +55,12 @@ $ npm install
 $ ELECTRON_MIRROR="https://npm.taobao.org/mirrors/electron/" npm install
 ```
 
-## License
-MIT
+## Licenses
+
+[MIT License](https://github.com/dat-land/dat-desktop/blob/master/LICENSE)
```

Comment:
Afaik many markdown implementations now support `[./LICENSE]` which would prevent us from having to rewrite it whenever that might change.